src/Slider/index.jsx

```diff
@@ -447,29 +447,31 @@ export default class Slider extends Component
             return;
         }
 
-        event.preventDefault();
-
         const { index } = event.target.dataset;
 
+        event.stopPropagation();
```

Comment:
I think we probably don't need this here

Reply:
Done